.github/workflows/label_prs.yml

```
@@ -1,13 +1,15 @@
 name: 'Pull Request Labeler'
 
 on:
-  pull_request_target:
+  pull_request_target: [open, assigned, path]
 
 jobs:
   labeler:
     permissions:
       contents: read
```

actionlint (1.7.12)

[error] "read write" is invalid as permission of scope "contents". available values are "read", "write", "none"

(permissions)